test/integration/goldens/asset/samples/snippets/generated/main/java/com/google/cloud/asset/v1/assetservice/create/SyncCreateComposeTransport.java

```diff
@@ -16,17 +16,17 @@
 
 package com.google.cloud.asset.v1.samples;
 
-// [START cloudasset_v1_generated_AssetService_Create_SetCredentialsProvider1_sync]
+// [START cloudasset_v1_generated_AssetService_Create_ComposeTransport_sync]
 import com.google.cloud.asset.v1.AssetServiceClient;
 import com.google.cloud.asset.v1.AssetServiceSettings;
 
-public class SyncCreateSetCredentialsProvider1 {
+public class SyncCreateComposeTransport {
 
   public static void main(String[] args) throws Exception {
-    syncCreateSetCredentialsProvider1();
+    syncCreateComposeTransport();
   }
 
-  public static void syncCreateSetCredentialsProvider1() throws Exception {
+  public static void syncCreateComposeTransport() throws Exception {
```

Comment:
This does not look correct to me, can we leave the SnippetGen code out of this PR for now? Then create a separate PR for the SnippetGen changes and ask @alicejli to review it?

Reply:
The only reason I included this in this PR was because there was samples being generated that use the `newHttpTransport()` code for services/clients that were not enabled for HTTP. Fixing this snippetgen logic removed generating the http code for non-enabled services.